apps/backend/src/lambdas/const/tdi34.const.ts

```diff
@@ -0,0 +1,10 @@
+export enum TDI34Enum {
+   BUCKET= 'bc-pcc-data-files-local',
+   LOCAL_PATH= 'tdi34/TDI34.TXT',
```

Comment:
I don't think we should have local file path or any logic for that. 

we can enable testing scripts to work with local stack.

Reply:
Agreed, this is actually a reference to the file in localstack. In order to develop locally I am calling `make upload-local-tdi(17||34)` which does rely on the static file in sample-files firectory at the root. `LOCAL_PATH` should be `LOCALSTACK_PATH` ?

Reply:
When we call the make command, we could also pass the local stack's path. 

This way anyone who wants to test and play with different data sets is not forced to use one file path and keep overwriting it. `tdi34/TDI34.TXT`